src/main/resources/application.yml

```diff
@@ -1,12 +1,33 @@
+version: v0.2.19
+
+---
+spring:
+  profiles: production
+  data:
+    rest:
+      return-body-on-create: true
+      return-body-on-update: true
+  jpa:
+    show-sql: false
 server:
-  port: ${port:8081}
+  port: ${port:80}
```

Comment:
Should the server config then not be indented as deep at the profiles at least for clarity?